DCode.Data/MetadataRepository/IApprovedApplicantRepository.cs

```diff
@@ -0,0 +1,10 @@
+﻿using DCode.Data.DbContexts;
+using System.Collections.Generic;
+
+namespace DCode.Data.MetadataRepository
+{
+    public interface IApprovedApplicantRepository
+    {
+        IEnumerable<approvedapplicant> GetApprovedApplicants();
```

Comment:
Can we not instead create a method which fetches the approved applicant only on the basis of the `Applicant_ID`? We need not fetch the data of the whole table into the service layer.

Reply:
This is addressed.